doc/source/serve/llm/serving-llms.rst

```diff
@@ -673,6 +673,29 @@ You can then specify the `bucket_uri` in the `model_loading_config` to point to
       name: llm_app
       route_prefix: "/"
 
+Configure tokenizer pool size
```

Comment:
I would put this under frequently asked questions and then remove it later once we completely switch to v1.